data/core/macros/special-notes.cfg

```diff
@@ -417,6 +480,34 @@ _"This unit has a defense cap on certain terrain types — it cannot achieve a h
 #deprecated 1 Use {NOTE_POISON} instead (note the singular), which generates a [special_note] tag.
 {INTERNAL:SPECIAL_NOTES_POISON}#enddef
 
+#define SPECIAL_NOTES_SHOCK
+#deprecated 1 Use {NOTE_SHOCK} instead (note the singular), which generates a [special_note] tag.
+{INTERNAL:SPECIAL_NOTES_SHOCK}#enddef
+
+#define SPECIAL_NOTES_UTBS_HEALS
+#deprecated 1 Use {NOTE_UTBS_HEALS} instead (note the singular), which generates a [special_note] tag.
+{INTERNAL:SPECIAL_NOTES_UTBS_HEALS}#enddef
+
+#define SPECIAL_NOTES_UTBS_CURES
+#deprecated 1 Use {NOTE_UTBS_CURES} instead (note the singular), which generates a [special_note] tag.
+{INTERNAL:SPECIAL_NOTES_UTBS_CURES}#enddef
+
+#define SPECIAL_NOTES_FORMATION
+#deprecated 1 Use {NOTE_FORMATION} instead (note the singular), which generates a [special_note] tag.
+{INTERNAL:SPECIAL_NOTES_FORMATION}#enddef
+
+#define SPECIAL_NOTES_DISENGAGE
+#deprecated 1 Use {NOTE_DISENGAGE} instead (note the singular), which generates a [special_note] tag.
+{INTERNAL:SPECIAL_NOTES_DISENGAGE}#enddef
+
+#define SPECIAL_NOTES_DAZE
+#deprecated 1 Use {NOTE_DAZE} instead (note the singular), which generates a [special_note] tag.
+{INTERNAL:SPECIAL_NOTES_DAZE}#enddef
+
+#define SPECIAL_NOTES_DISTRACT
+#deprecated 1 Use {NOTE_DISTRACT} instead (note the singular), which generates a [special_note] tag.
+{INTERNAL:SPECIAL_NOTES_DISTRACT}#enddef
```

Comment:
As these macros weren't in core before, I don't think they need deprecated versions.

Reply:
Huh, I am not sure, but it can't hurt to have them.